index.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Simple-singolo</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="header">
+        <div class="header__burger header__burger--not-active"></div>
+        <div class="header__shadow"></div>
+        <img src="assets/images/logo.png" alt="logo" class="header__logo">
+        <nav>
+            <ul class="navigation">
+                <li class="navigation__item"><a href="#" class="navigation__link navigation__link--active">HOME</a></li>
+                <li class="navigation__separator">&middot;</li>
+                <li class="navigation__item"><a href="#services" class="navigation__link">SERVICES</a></li>
+                <li class="navigation__separator">&middot;</li>
+                <li class="navigation__item"><a href="#portfolio" class="navigation__link">PORTFOLIO</a></li>
+                <li class="navigation__separator">&middot;</li>
+                <li class="navigation__item"><a href="#" class="navigation__link">ABOUT</a></li>
+                <li class="navigation__separator">&middot;</li>
+                <li class="navigation__item"><a href="#" class="navigation__link">CONTACT</a></li>
+            </ul>
+        </nav>
+    </header>
+    <div class="decoration-line"></div>
+    <section class="slider">
+        <div class="slider__line">
+            <img src="assets/images/slide1.jpg" alt="photo1" class="slider__photo">
+            <img src="assets/images/slide2.jpg" alt="photo2" class="slider__photo">
+        </div>
+        <img src="assets/images/arrow-left.png" alt="arrow-left" class="slider__arrow-left">
+        <img src="assets/images/arrow-right.png" alt="arrow-right" class="slider__arrow-right">
```

Comment:
Extract please SVG icon. Actually you can extract only one arrow icon and duplicate it using css transformation to rotate it.